app/ui-react/packages/atlasmap-adapter/src/DataMapperAdapter.tsx

```diff
@@ -76,6 +76,7 @@ export const DataMapperAdapter: React.FunctionComponent<IDataMapperAdapterProps>
     console.log('Atlasmap document', JSON.stringify(externalDocument));
     return (
       <AtlasmapProvider
+        logLevel={'info'}
```

Comment:
Should probably take this out, might be too noisy.

Reply:
Ah ok, yea I just added it because it throws a TS error that I otherwise couldn't seem to get around. Maybe we could make it an optional property or have a 'silent' type option?

Reply:
`warn` or `silent` should work.